html-block-and-inline/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>html-block-and-inline</title>
+</head>
+<body>
+  <h1>Blue Americana Snuggie&reg; Blanket</h1>
+
```

Comment:
I would remove any extra space from between the elements. It is best to avoid whitespace in the code.